.github/workflows/build.yml

```diff
@@ -419,3 +419,63 @@ jobs:
         with:
           name: build
           path: build/
+
+  # === Platform Upload job ===
+  platform_upload:
+    name: Platform Upload
+    runs-on: ubuntu-20.04
+    #if: contains(fromJSON('["refs/heads/master", "refs/heads/beta", "refs/heads/release", "refs/heads/LTS"]'), github.ref)
+    env:
+      STUBFILE: deployments/platform/stub.yaml
+      CHANNEL: ${{ github.event.pull_request.base.ref }}
+
+    # === Platform Upload Steps ===
+    steps:
+
+      - # === Checkout Code ===
+        name: Checkout Code
+        uses: actions/checkout@v2
+
+      - # === Generate Stub ===
+        name: Generate Stub
+        shell: bash
+        run: |
+          VERSION="`head -n 1 version.txt`"
+          AUTHOR=00000000-0000-0000-0000-000000000000
+          sed -ie "s/\$CHANNEL/$CHANNEL/;" $STUBFILE
+          sed -ie "s/\$VERSION/$VERSION/;" $STUBFILE
+          sed -ie "s/\$AUTHOR/$AUTHOR/;" $STUBFILE
+
+      - # === Upload Stub as Artifact ===
+        name: Upload Stub as Artifact
+        uses: actions/upload-artifact@v2
+        with:
+          name: platform-stub.yaml
+          path: ${{ env.STUBFILE }}
+
+      - # === Checkout TheHomeRepot ===
+        name: Checkout TheHomeRepot
+        uses: actions/checkout@v2
+        with:
+          repository: ActiveState/TheHomeRepot
+          path: TheHomeRepot
+
+      - # === Upload Stub ===
+        name: Upload Stub
+        if: false
+        shell: bash
+        run: |
+          apt-get install -y pipenv
```

Comment:
Note we should be dogfooding the platform, meaning pip is banned. Is this what LE/BE does?

Reply:
@icanhasmath Are those docs up to date?

Reply:
> @icanhasmath Are those docs up to date?

All the pipenv stuff is out of date.

Reply:
The adding ingredient documentation here should be mostly up to date:
https://handbook.activestate.build/teams/build-engineering/ingredients/add-new-ingredient/#import-ingredients-manually-using-be-tooling
It still references 
```
                 sortable_version:
                  - '0'
```
Which is not needed anymore.